bigtable/tests/system.py

```diff
@@ -79,19 +80,19 @@ def setUpModule():
         credentials = EmulatorCreds()
         Config.CLIENT = Client(admin=True, credentials=credentials)
     else:
-        Config.CLIENT = Client(admin=True)
+        Config.CLIENT = Client(project=PROJECT_ID, admin=True)
```

Comment:
Shouldn't be needed:  by default, the client looks it up.

Reply:
Removed